### ISSUE TITLE
Does IntPSQ.atMostView return an ordered list or not?

### ISSUE THREAD

**Issue by mitchellwrosen** [jaspervdj/psqueues#57]
Hi, I think the documentation of `IntPSQ.atMostView` is self-contradictory (emphasis mine):

> **Return a list of elements ordered by key** whose priorities are at most pt, and the rest of the queue stripped of these elements. **The returned list of elements can be in any order**: no guarantees there.

Perhaps the docs just need to be adjusted?

Thanks!

**Comment by jaspervdj** [comment]
Good catch!  I think the _original intent_ of the second sentence is to stress that they are not ordered by priority in any sense.  Would you be willing to add a PR to reword this?
